src/main/java/hudson/plugins/plot/PlotAction.java

```diff
@@ -56,6 +59,11 @@ public boolean hasPlots() throws IOException {
     }
 
     // called from PlotAction/index.jelly
+    public Job<?, ?> getProject() {
```

Comment:
So, if you have a getJob that returns the project, then your Jelly can simple do it.job and keep compatibility with external consumers.